app/src/main/res/layout/exploration_fragment.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout
+  xmlns:android="http://schemas.android.com/apk/res/android">
+  <data>
```

Comment:
I don't believe we need this data section if it doesn't have anything in it. The outer layout tag is sufficient for generating the data binding subclass.

Reply:
Thanks. I was not aware of this.
Removed `<data>` tag